### ISSUE TITLE
[LgThinqAirConditioner] Error discovering devices Error: Request failed with status code 400

### ISSUE THREAD

**Issue by fleeeecy** [sman591/homebridge-lg-thinq-ac#53]
**LG Air Conditioner Model**
LP1419IVSM

**Describe the bug**
Error in the log : [LgThinqAirConditioner] Error discovering devices Error: Request failed with status code 400
Unit won't show up in Home app

**Symptom check**
* **Does your unit appear in the Home app?** No
* **Is the current on/off state, temperature, and mode being reported?** No
* **Does your unit respond to the controls?** No

**Relevant logs**
2020-11-27, 4:26:01 PM [LgThinqAirConditioner] Initializing LgThinqAirConditioner platform...
2020-11-27, 4:26:07 PM [LgThinqAirConditioner] Already logged into ThinQ
2020-11-27, 4:26:33 PM [LgThinqAirConditioner] Error discovering devices Error: Request failed with status code 400

I tried a full reboot, no change in behavior. This unit is attached to a canadian LG account, so CA and en-CA are used. Any hint would help Thanks!

**Comment by sman591** [comment]
Added a feature to catch this + prompt to re-auth. Could you try:

1. Update to the latest plugin version
2. Restart homebridge
3. Follow the instructions printed to the log?

**Comment by fleeeecy** [comment]
Hi, I just installed v1.14.0 and tried doing the config again. I'm getting the same error unfortunately. 

I'm not seeing a prompt to re-auth as you mentioned. 

2021-04-20, 10:17:18 AM [LgThinqAirConditioner] Already logged into ThinQ
2021-04-20, 10:17:30 AM [LgThinqAirConditioner] Error discovering devices Error: Request failed with status code 400

**Comment by sman591** [comment]
I see. Could you try manually clearing the "access token" and "refresh token" fields in the config, restart home bridge, and then follow the login steps again? (followed by a restart at the end)

**Comment by fleeeecy** [comment]
Hi, thanks for trying to help! Much appreciated! 

I cleared the "access token" and "refresh token" fields in the config, clicked Save Changes then restarted Homebridge (Save Changes does it automatically). Unfortunately, I'm not seeing much difference. 

2021-04-22, 1:05:59 PM [LgThinqAirConditioner] Initializing LgThinqAirConditioner platform...
-
2021-04-22, 1:06:10 PM [LgThinqAirConditioner] Initiating auth with provided redirect URL
2021-04-22, 1:06:23 PM [LgThinqAirConditioner] Error discovering devices Error: Request failed with status code 400

(rebooted the Raspberry Pi3 Homebridge is running on, to see if anything changes)

2021-04-22, 1:08:01 PM [LgThinqAirConditioner] Initializing LgThinqAirConditioner platform...
-
2021-04-22, 1:08:23 PM [LgThinqAirConditioner] Already logged into ThinQ
2021-04-22, 1:08:34 PM [LgThinqAirConditioner] Error discovering devices Error: Request failed with status code 400

**Comment by sman591** [comment]
After clearing "access token" and "refresh token", did you follow the login URL again to replace the "redirected URL" field?

https://github.com/sman591/homebridge-lg-thinq-ac/blob/master/README.md#installation-guide

**Comment by fleeeecy** [comment]
> After clearing "access token" and "refresh token", did you follow the login URL again to replace the "redirected URL" field?
> 
> https://github.com/sman591/homebridge-lg-thinq-ac/blob/master/README.md#installation-guide

I did redo the login procedure, yes. 

Just to be sure, I did it again just now, and I'm getting the same result unfortunately. Is there a more "verbose" log I could provide you possibly?

2021-04-23, 11:12:27 AM [LgThinqAirConditioner] Initiating auth with provided redirect URL
2021-04-23, 11:12:39 AM [LgThinqAirConditioner] Error discovering devices Error: Request failed with status code 400
